src/theme/icons/angleDown.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Font Awesome Free 6.1.2 by @fontawesome - https://fontawesome.com
+ * License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
+ * Copyright 2022 Fonticons, Inc.
+ */
+
+import {
+  IconDefinition,
+  IconName,
+  IconPrefix,
+} from '@fortawesome/fontawesome-svg-core';
+
+export const width = 512;
+export const height = 448;
```

Comment:
@Mnau-mnau I think you switched those two numbers. Icons now seem to be off-center. Other icons seems to be ok.

Reply:
you're right, fixed